electora_py/api.py

```diff
@@ -1,11 +1,33 @@
-from nucypher.characters.lawful import Bob, Ursula
+from typing import Optional, Dict
+from gql import gql, Client
+from gql.transport.aiohttp import AIOHTTPTransport
+from nucypher_core import ferveo
+
+from nucypher.characters.lawful import Ursula, Bob
 from nucypher.cli.utils import connect_to_blockchain
 from nucypher.utilities.emitters import StdoutEmitter
-from nucypher_core import ferveo
+
+# Networks
```

Comment:
Can we please avoid using comments unless they're explaining something that's not obvious?

Reply:
If you'd prefer to not have a comment here, I'll remove it.  I think of this as a logical break appropriate for the current development stage.

Reply:
I loathe comments. They get chucked in, don't get maintained  and just add bloat to a codebase. Unless they're explaining something that's tricky to get your head round, they should be avoided. (Even then, refactor out to a function and give it a decent docstring is often a better option).